base/file.jl

```diff
@@ -341,37 +353,47 @@ Obtain the path of a temporary directory (possibly shared with other processes).
 tempdir()
 
 """
-    tempname()
+    tempname(;prefix="jl_")
```

Comment:
You can replace `"jl_"` with `$temp_prefix` to make sure it's in sync with the corresponding constant.